frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -50,11 +63,24 @@ const LineAreaBarChart = ({
     goalColor: getColor("text-medium"),
   };
 
+  const palette = { ...colors, ...instanceColors };
```

Comment:
Actually, there is a `getColor` function, but it isn't compatible with APIs in 
https://github.com/metabase/metabase/blob/cc589cf0f6a3f4dfe8e22aee2c1f21a8a595ddd0/frontend/src/metabase/lib/colors/palette.ts#L83
since that expects a whole color palette to be passed not a function.

That's why I construct the whole palette with user colors instead.

Reply:
Can we unify this to avoid having different approaches?